src/core/api/credentials/credentials.api.ts

```diff
@@ -52,3 +54,23 @@ export async function createCredentialWithRecipient(payload: CredentialRecipient
 export async function sendCredentials(payload: SendCredentialsReq): Promise<SuccessRes> {
   return (await post<SuccessRes>('credentials/notify/via-schema', payload)).data;
 }
+
+export async function importCSVFile(payload: ImportFileReq): Promise<ImportFileRes> {
+  const { file, ...rest } = payload;
+  const formData = new FormData();
+  formData.append('file', file);
+  for (const key in rest) {
+    if (rest[key] !== undefined) {
+      formData.append(key, String(rest[key]));
+    }
+  }
```

Comment:
You can write a util for this ( and reuse it later on )

Reply:
oh yeah thanks a lot :+1: